### DISCUSSION TITLE
convert single-cell Hi-C to bulk Hi-C

### DISCUSSION THREAD

**Discussion by BenxiaHu** [open2c/cooler#388] [Q&A] [answered]
Hello,
    cooler can create a cool file for scHi-C (https://cooler.readthedocs.io/en/stable/schema.html#single-cell-single-resolution). it may work. But it require to provide all informations for each cell.

   I was just wondering whether cooler can merge all single-cell cool files togeter to create a psudo-bulk cool file.  
   https://cooler.readthedocs.io/en/stable/api.html#cooler.merge_coolers
  
Best,

**Comment by nvictus** [comment, marked as answer]
Hi @BenxiaHu. The answer is yes!

Single-cell `.scool` files contain a cooler data collection for each cell, defined by the unique URI `example.scool::cells/{cell_id}` for each cell ID. Each one of those points to a standard cooler collection, so you can feed them all to `merge_coolers` to produce a pseudo-bulk output file.

You can also do it via the CLI:

```sh
$ cooler merge `cooler ls example.scool` pseudobulk.cool
$ cooler zoomify --balance pseudobulk.cool  # make multi-resolution if desired
```
